lighthouse-core/lib/manifest-parser.js

```diff
@@ -37,21 +37,21 @@ const ALLOWED_ORIENTATION_VALUES = [
  */
 function parseString(raw, trim) {
   let value;
-  let debugString;
```

Comment:
seems like we want the `manifest-parser` still returning something like `debugString`? `explanation` seems too generic for what is always indicating an error/warning. Gatherers that use it would still only use the standard thing

Reply:
Originally in `manifest-parser` it was `warning`, which got switched to `debugString`, which was then spread throughout Lighthouse cause why not, all in #102 :)

Reply:
> explanation seems too generic for what is always indicating an error/warning. 

sounds like you're upset with our decision to use `explanation` period :)

I'm not 100% sure what you're advocating for, is it for this PR to just remove the TODO and fix the tests then since we don't want to pursue it?

Reply:
> sounds like you're upset with our decision to use `explanation` period :)

haha, well, `explanation` on an audit is a little confusing in isolation (we really should add a jsdoc description to it in `audit.d.ts` :), but it comes in the context of a failing audit.

In this case the only indication that there was an issue is the existence of the string. I meant *just* for `manifest-parser.js`, it's not in the gatherer or audit pipelines, it's just a lib file, so it seems fine to have a more descriptive property name specific to that file.

Reply:
looked and within manifest-parser, i'd say that, semantically, all these properties are really `warning`s. Except for the `errorMessage` at the end about invalid JSON

So i'd be in favor of changing the manifestparser explanation->warning for all items and errorMesssage on the toplevel. and also tweaking manifest-values.js to use these terms as well.

Reply:
@brendankenny are you on board with `warning` as well? I think it reads rather well now

Reply:
> are you on board with warning as well?

yes, sounds good to me

> (aside from the fact that it's warning = 'ERROR: 😆)

lol yes, that's terrible